src/js/templates/maps/legend/layer-legend.html

```diff
@@ -0,0 +1,2 @@
+<div class="layer-legend__layer-name"><%= name %></div>
+<div class="<%= classNames.palette %>">palette</div>
```

Comment:
Is this just a placeholder for work in progress?

Reply:
Yes! The implementation is in https://github.com/NCEAS/metacatui/pull/2445 and https://github.com/NCEAS/metacatui/pull/2446. Sorry that wasn't very clear